templates/Silk.NET.Templates/Silk.NET.Templates.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageType>Template</PackageType>
+    <PackageVersion>1.0</PackageVersion>
+    <PackageId>Silk.NET.Templates</PackageId>
+    <Title>Silk.NET Project Templates</Title>
+    <Authors>.NET Foundation and Contributors</Authors>
+    <Description>A package with template projects for Silk.NET.</Description>
+
+    <TargetFramework>netstandard2.0</TargetFramework>
+
+    <IncludeContentInPack>true</IncludeContentInPack>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <ContentTargetFolders>content</ContentTargetFolders>
+    <NoWarn>$(NoWarn);NU5128</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="templates\**\*" Exclude="templates\**\bin\**;templates\**\obj\**" />
+    <Compile Remove="**\*" />
```

Comment:
How about `<EnableDefaultCompileItems>false</EnableDefaultCompileItems>` ?

Reply:
Yeah, that definitely looks nicer. Thanks for the suggestion! :)